pytorch_lightning/accelerators/accelerator.py

```diff
@@ -127,7 +127,9 @@ def optimizer_step(self, optimizer, batch_idx, opt_idx, lambda_closure):
             optimizer_closure=lambda_closure,
             on_tpu=False,  # TPUAccelerator class sets this as True
             using_native_amp=using_native_amp,
-            using_lbfgs=is_lbfgs
+            using_lbfgs=is_lbfgs,
+            *args,
```

Comment:
personally, I would not use it... but should be fine :]

Reply:
I followed Will advice. But it makes sense to carry arguments over.